### ISSUE TITLE
Max connections error PostgreSQL on Heroku

### ISSUE THREAD

**Issue by Jhony0311** [prismagraphql/prisma#2680]
**Describe the bug**
With a project locally, set up with a self hosted database 
After I do `docker-compose up -d` I start getting max connection errors. Inconsistently, could succeed `prisma deploy` but still not working.

**To Reproduce**
1. Set up a Prisma project with a self hosted database on Heroku using [Heroku Postgres Hobby Plan](https://elements.heroku.com/addons/heroku-postgresql)
2. Lift your Docker image locally and try to deploy locally by doing `prisma deploy`
3. Check docker container errors with `docker logs yourContainerName`
```
SEVERE: Connection error: 
org.postgresql.util.PSQLException: FATAL: too many connections for role "ximdxudtymukbs"
at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
at org.postgresql.Driver.makeConnection(Driver.java:452)
at org.postgresql.Driver.connect(Driver.java:254)
at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
```
And not able to get my local server running.

**Versions (please complete the following information):**
 - OS: Mac OS High Sierra 10.13.4 (17E202) / MacBook Pro (17-inch, Early 2011)
 - `prisma` CLI: prisma/1.10.2 (darwin-x64) node-v8.10.0
 - Prisma Server: 1.10
 - `graphql-cli v2.16.3`

**Additional context**
It work perfect with the next plan Standard 0 with 120 connection limit, in a simple usage it get to use 43 connections max and 22 connections min.

**Comment by marktani** [comment]
Hey @Jhony0311, thanks a lot for bringing this up!
We pushed a fix to `1.13-alpha` for this here: https://github.com/prismagraphql/prisma/pull/2717.

**Comment by marktani** [comment]
This is fixed on `1.12` which has been released today.
